src/lib/models/versions.ts

```diff
@@ -24,6 +25,15 @@ export type VersionMetadata = {
   objects: string[];
 };
 
+export type ModLinkData = {
```

Comment:
Please rename to `ModArchData`

Reply:
https://github.com/satisfactorymodding/smr-frontend/pull/47/commits/9c0893e0353c97b347f9f0152ec590c614d255ac